src/main/java/dao/AddToStorage.java

```diff
@@ -0,0 +1,5 @@
+package dao;
+
+public interface AddToStorage {
```

Comment:
Actually, you can create one interface for all crud operations, but if you want to separate them, then let's have one class that will implement them. You can even create a general interface that will inherit all small ones